user_sync/rules.py

```diff
@@ -743,14 +746,21 @@ def update_umapi_users_for_connector(self, umapi_info, umapi_connector):
         if self.will_process_strays:
             self.add_stray(umapi_info.get_name(), None)
 
+        if self.options['adobe_group_filter']:
```

Comment:
We're probably safer changing this to `if self.options.get('adobe_group_filter') is not None`.